tests/ci/labels.go

```diff
@@ -17,6 +17,9 @@ var Day2 = Label("day2")
 // destroy
 var Destroy = Label("destroy")
 
+// import
+var ImportLabel = Label("tf-import")
```

Comment:
may be use feature-import name - similar to our idp and mp cases.

Reply:
changed